src/common/docker-compose/converter.ts

```diff
@@ -123,7 +123,9 @@ export class ComposeConverter {
           }
         }
       } else {
-        build.args = compose_build.args;
+        for (const [key, value] of Object.entries(compose_build.args as Dictionary<string>)) {
+          build.args[key] = value.toString();
```

Comment:
Is the `toString` needed if the Dictionary is already casting everything to a string?

Reply:
We shouldn't use `Dictionary<string>` here because what's actually being passed in is a `Dictionary<any>`. The `as` is just telling TS that everything is okay and it'll only see string values, but this isn't actually casting the values. This compose file 

```yml
version: 3
services:
  test-service:
    build:
      args:
        API_VERSION: $API_VERSION
        EMPTY:
```

will cause this to explode because of what I mentioned above. We'll need a little more logic here to handle all of the cases that a user could throw at us, for example:
```ts
build.args[key] = !!value ? value.toString() : value;
```

We should include a test for an empty arg here too

Reply:
Sounds good. I'll fix it.